lang/ruby/lib/avro/schema.rb

```diff
@@ -38,6 +38,8 @@ class Schema
 
     DEFAULT_VALIDATE_OPTIONS = { recursive: true, encoded: false }.freeze
 
+    DECIMAL_LOGICAL_TYPE = 'decimal'.freeze
```

Comment:
Perhaps not a big deal, but this does not seem to be used anywhere except in the derived FixedSchema. Would it make sense to move it there instead of the base class?

Reply:
It is used in both the FixedSchema and the BytesSchema:
- BytesSchema: https://github.com/apache/avro/pull/1082/files/7ea99640fd96204dcb217133da6c3c6394ef043c#diff-410be18ca388364fc161738379922d6c2df9b7f392a58203cc051a8038f962caR505
- FixedSchema: https://github.com/apache/avro/pull/1082/files/7ea99640fd96204dcb217133da6c3c6394ef043c#diff-410be18ca388364fc161738379922d6c2df9b7f392a58203cc051a8038f962caR539

Reply:
👍 Sorry, missed the other usage